oidc-service-impl/src/main/java/io/mosip/esignet/services/AuthorizationServiceImpl.java

```diff
@@ -212,7 +212,7 @@ public AuthResponseV2 authenticateUserV3(AuthRequestV2 authRequest, HttpServletR
         if(!CollectionUtils.isEmpty(captchaRequired) &&
                 authRequest.getChallengeList().stream().anyMatch(authChallenge ->
                         captchaRequired.contains(authChallenge.getAuthFactorType().toLowerCase()))) {
-            authorizationHelperService.validateCaptchaToken(authRequest.getCaptchaToken());
+            captchaHelper.validateCaptcha(authRequest.getCaptchaToken());
```

Comment:
Lets not make any change here, only in the authorizationHelperService lets change the wiring